app/stac_api/models.py

```diff
@@ -242,6 +242,9 @@ def __str__(self):
     def save(self, *args, **kwargs):  # pylint: disable=signature-differs
         # It is important to use `*args, **kwargs` in signature because django might add dynamically
         # parameters
+        if self.description == "":
+            logger.debug("transforming empty description string into None for database storage")
+            self.description = None
```

Comment:
This is too hacky and would probably breaks the API, the field needs to be changed instead to have its default to `None` like for the Asset:

```
# here we need to set blank=True otherwise the field is as required in the admin interface
description = models.TextField(blank=True, null=True, default=None)
```

And the description needs to be changed in the admin interface like for `AssetAmin`
```python
def save_model(self, request, obj, form, change):
        if obj.description == '':
            # The admin interface with TextArea uses empty string instead
            # of None. We use None for empty value, None value are stripped
            # then in the output will empty string not.
            obj.description = None

        super().save_model(request, obj, form, change)
```